component-server-parent/component-server/src/test/java/org/talend/sdk/component/server/front/ActionResourceImplTest.java

```diff
@@ -345,4 +346,65 @@ void executeFailWithBadEncrypted() {
                 response.readEntity(ErrorPayload.class).getDescription());
     }
 
+    @Test
+    void executeDynamicDependencies() {
+        HashMap<String, String> params = new HashMap();
+        List<String> drivers;
+        //
+        params.put("configuration.url", "zorglub://vault:8200/fail");
+        params.put("configuration.driver", "zorglub");
+        params.put("configuration.query", "SELECT 1;");
+        Response response = base
+                .path("action/execute")
+                .queryParam("type", "dynamic_dependencies")
+                .queryParam("family", "jdbc")
+                .queryParam("action", "jdbc-deps")
+                .request(APPLICATION_JSON_TYPE)
+                .post(Entity.entity(params, APPLICATION_JSON_TYPE));
+        assertEquals(200, response.getStatus());
+        drivers = response.readEntity(List.class);
+        assertEquals(3, drivers.size());
+        assertEquals(
+                "[org.zorglub:zorglub-client:1.0.0, org.bouncycastle:bcpkix-jdk15to18:jar:4.5.8, org.json:json:jar:2.0.0]",
```

Comment:
So it can either use ":jar:" or omit it?

Reply:
Yes, it's intended to be parsed by the `Mvn.locationToMvn()` static method in tdi-studio-se.